static/app/views/performance/cache/referrers.ts

```diff
@@ -1,4 +1,10 @@
 export enum Referrer {
-  LANDING_CACHE_HIT_MISS_CHART = `api.performance.cache.landing-cache-hit-miss-chart`,
-  LANDING_CACHE_THROUGHPUT_CHART = `api.performance.cache.landing-cache-throughput-chart`,
+  LANDING_CACHE_HIT_MISS_CHART = `'api.performance.cache'.landing-cache-hit-miss-chart`,
+  LANDING_CACHE_THROUGHPUT_CHART = `'api.performance.cache'.landing-cache-throughput-chart`,
+  LANDING_CACHE_TRANSACTION_LIST = `'api.performance.cache'.landing-cache-transaction-list`,
+
+  SAMPLES_CACHE_METRICS_RIBBON = `'api.performance.cache'.samples-cache-metrics-ribbon`,
+  SAMPLES_CACHE_TRANSACTION_DURATION = `'api.performance.cache'.samples-cache-transaction-duration`,
+  SAMPLES_CACHE_SPAN_SAMPLES = `'api.performance.cache'.samples-cache-span-samples`,
+  SAMPLES_CACHE_SPAN_SAMPLES_TRANSACTION_DURATION = `'api.performance.cache'.samples-cache-span-samples`,
```

Comment:
@gggritso looks like the prefix thing isn't even allowed anymore from lint rules lol
https://github.com/getsentry/sentry/commit/9952d7f6bf31da06423a8032f8ae2b3de1c2bc2e#diff-f70e031a6ba39caa59a58e9a1ae89c8d05cb113d8462275980c7b9c174d4127eL1

Reply:
Well, there you go!